src/okta_inspector/engine.py

```
@@ -0,0 +1,195 @@
+                all_findings.extend(findings)
+            except Exception as e:
+                logger.error("Error in %s analyzer: %s", analyzer.name, e)
+        return all_findings
+
+    def _run_reporters(self, findings: list[ComplianceFinding], data: OktaData) -> None:
+        reporters = get_reporters()
+        for reporter in reporters:
+            logger.info("Generating %s report...", reporter.display_name)
+            try:
+                reporter.generate(findings, data, self._output)
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)